rfcs/convergence/authoring-stories.md

```diff
@@ -433,6 +436,46 @@ export const Default = (props: PopoverProps) => (
 Public stories should only contain code, which is useful for users to see after clicking on “Show code” in documentation.
 Extra markup (e.g., container with CSS styles) can be added via [Decorators](https://storybook.js.org/docs/react/writing-stories/decorators).
 
+### 10. Internal stories for testing
+
+As mentioned in for E2E testing, we should ensure maximum coverage for all publicly viewable stories by our consumers.
+For more complex scenarios that need to be tested we should make sure that stories exist for E2E tests but should not be
+easily accessible publicly.
+
+Storybook has proposed a feature for this in [storybookjs/storybook#9209](https://github.com/storybookjs/storybook/issues/9209)
+which will configure stories to exist in deeplink URL format, but do not appear in the nav tree or the docs page. As stated in the issue,
+we can workaround before the release of this feature by modifying `manager-head.html` and set `display:none`for all
```

Comment:
hmm just a note, AFAIR storybook doesn't support inheritance when `manager-head.html` is used -> thus if this is being set in root `./storybook` and some package would like to use some specific manager overrides this would break the flow

Reply:
This measure is only to ensure that we don't make it easy to see these stories, for users in the public docsite. therefore this config in `react-components` should be the only one needed, although I could be missing some long term storybook composition details you might be planning

Reply:
#19040 finally introduces a concept of production by setting NODE_ENV, so we can achieve the same after that PR by doing custom story loading, which can also be messy.... I don't mind which one, but this proposal has already been blocked multiple times in the last few months and blocks us from testing meaningful edge cases in the browser